app/containers/EmojiPicker/index.tsx

```diff
@@ -20,6 +20,8 @@ const EmojiPicker = ({
 	searchedEmojis = []
 }: IEmojiPickerProps): React.ReactElement | null => {
 	const { colors } = useTheme();
+	const [windowWidth, setWindowWidth] = useState(0);
```

Comment:
I disagree on this naming, because it's not the window's width.
It's the component's width.
EmojiPicker's, specifically.

I'd say `width` is enough, since the relevance of it is inside EmojiCategory only.
It doesn't need to understand where the width is coming from.
It could be `parentWidth`, if you wanted to be specific.
WDYT?